📦 NEW: add Alfred snippet collection

davidsword opened the pull request:
I've recently (15 min ago) adopted Emoji-Log. Great work on this (and everything you do), thank you!

First thing I did was add this to Alfred. Thought to share.

A snippet collection for Alfred PowerPack users. https://www.alfredapp.com/powerpack/

Intentionally, the "Name" column of the collection does not start with
an emoji. Alfreds Snippet browser doesn't use fuzzy find, it appears
to work off of first letters. Having the emojis as the lead character
makes the words hard to browse this way.

With auto expand turned on, the following keywords will expand to:

| Keyword | Snippet |
| --- | --- |
| `;new` | 📦 NEW: |
| `;improve` | 👌 IMPROVE: |
| `;fix` | 🐛 FIX: |
| `;release` | 🚀 RELEASE: |
| `;doc` | 📖 DOC: |
| `;test` | ✅ TEST: |

Screeny:

![sdfgsdgsfdgfdg](https://user-images.githubusercontent.com/1996370/55038528-5b9f2080-4fde-11e9-853f-6e6a6d5975db.gif)

⚠️ need an image probably, and I anticipate some rewording in the readme. Making a draft for now.

davidsword committed:
📦 NEW: add Alfred snippet collection

A snippet collection for Alfred PowerPack users.

Intentionally, the "Name" column of the collection does not start with
an emojji. Alfreds Snippet browser doesn't use fuzzy find, it appears
to work off of first letters. Having the emojjis as the lead chatacter
makes the words hard to browse this way.

With auto expand turned on, the following keywords will expand to:

;new = 📦 NEW:
;improve = 👌 IMPROVE:
;fix = 🐛 FIX:
;release = 🚀 RELEASE:
;doc = 📖 DOC:
;test = ✅ TEST:

ahmadawais commented on README.md:
```diff
@@ -241,6 +241,33 @@ If your repo uses `EMOJI-LOG` then you can add any of the following badges to yo
+| `;doc` | 📖 DOC: |
+| `;test` | ✅ TEST: |
+
+To edit the `;` prefix to your preferred expansion flag, double click right click the Emoji-Log Collection in _Alfred Settings » Features » Snippets_.
```
I use the same exact setup for TextExpander but with a `g` prefix.

| Keyword | Snippet |
| --- | --- |
| `;gnew` | 📦 NEW: |
| `;gimp` | 👌 IMPROVE: |
| `;gfix` | 🐛 FIX: |
| `;grlz` | 🚀 RELEASE: |
| `;gdoc` | 📖 DOC: |
| `;gtst` | ✅ TEST: |


Can you consider to change the snippets to reflect that? It's easier to remember and namespaced snippets since `g` stands for `git`.

davidsword replied:
Yes great point! I missed that that was the naming convention in the shell functions. Fixed in https://github.com/ahmadawais/Emoji-Log/pull/14/commits/25e92557cccdf4ba27c8ed829c870c4f58f56efd

ahmadawais commented:
Welcome here @davidsword awesome PR.

Just requested a change. 

Looking forward, peace! ✌️

davidsword committed:
👌 IMPROVE: use "g" namespace w/ shorter keywords

It's easier to remember and namespaced snippets since g stands for git.

Shorter keywors match shell function naming.

davidsword committed:
🐛 FIX: format TODO properly for todo bot

the bot expects the flag at the begining of a line, not the end.

ahmadawais commented:
@davidsword let's finish this PR :)

davidsword commented:
@ahmadawais would you like to create a matching bowler hat graphic for the heading? or should I just remove the image placeholder for now?

ahmadawais commented:
@davidsword I'll take care of the graphics. Let me know when you're ready.

Commit:
📦 NEW: Icon

davidsword commented:
Thanks, looks great!

@ahmadawais all good here! 🚢

ahmadawais reviewed (approved):
Looks good to me! 🔥

ahmadawais commented:
Thank you for your awesome contribution, @davidsword! 💯

davidsword commented:
@ahmadawais Thank you for creating an awesome repo to contribute too!

davidsword commented:
Just adding a note here that I personally had troubles memorizing the short hands, for the life of me I couldn't fit `gtst`/`grlz` into my brain. I ended up doing the following:

![Screen Shot 2019-04-18 at 10 51 39 AM](https://user-images.githubusercontent.com/1996370/56380927-53df3000-61c8-11e9-84a6-948e8988b1c5.png)

This allowed a easy snippet keyword search of `git `

![Screen Shot 2019-04-18 at 10 55 01 AM](https://user-images.githubusercontent.com/1996370/56381264-3b234a00-61c9-11e9-98e4-e12183f34c92.png)

I know I can browse `emoj ` and get a link to the pack, but that was one extra step I didn't want to have

![Screen Shot 2019-04-18 at 11 01 19 AM](https://user-images.githubusercontent.com/1996370/56381315-5beb9f80-61c9-11e9-9642-8193bf5efc6e.png)

No action required on this, just adding it in case anyone else runs into something similar, but probably boils down to personal preference.
